apps/api/docs/api-test-cases.md

```diff
@@ -30,7 +30,7 @@ Before running the test cases, make sure you have the following prerequisites in
 
    1. Open Postman.
    2. Click on the `Import` button.
-   3. Select the `apps/api/osmo-notify.postman_collection.json` file.
+   3. Select the `apps/api/osmoX.postman_collection.json` file.
```

Comment:
You can fix the casing for this as `OsmoX.postman_collection.json`

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b